observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc

```diff
@@ -0,0 +1,146 @@
+
+[NOTE]
+====
+A major difference between the Loki Operator and Loki itself lies in inheritance handling. Loki only copies `default_resource_attributes_as_index_labels` to tenants by default, while the Loki Operator applies the entire global configuration to each tenant in `openshift-logging` mode.
+====
+
+Within `LokiStack`, attribute mapping configuration is managed through the `limits` setting:
+
+[source,yaml]
+----
+# [...]
+spec:
+  limits:
+    global:
+      otlp: {} # <1> 
+    tenants:
+      application:
+        otlp: {} # <2> 
+----
+<1> Global OTLP attribute configuration.
+<2> OTLP attribute configuration for the `application` tenant within `openshift-logging` mode.
+
+[NOTE]
+====
+Both global and per-tenant OTLP configurations can map attributes to stream labels or structured metadata. At least _one stream-label_ is required to save a log entry to Loki storage, so ensure this configuration meets that requirement.
+====
+
+Stream labels derive only from resource-level attributes, which the `LokiStack` resource structure reflects:
+
+[source,yaml]
+----
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          resourceAttributes:
+          - name: "k8s.namespace.name"
+          - name: "k8s.pod.name"
+          - name: "k8s.container.name"
+----
+
+Structured metadata, in contrast, can be generated from resource, scope or log-level attributes:
+
+[source,yaml]
+----
+# [...]
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          # [...]
+        structuredMetadata:
+          resourceAttributes:
+          - name: "process.command_line"
+          - name: "k8s\\.pod\\.labels\\..+"
+            regex: true
+          scopeAttributes:
+          - name: "service.name"
+          logAttributes:
+          - name: "http.route"
+----
+
+[TIP]
+====
+Use _regular expressions_ by setting `regex: true` for attributes names when mapping similar attributes in Loki. 
+====
+
+[IMPORTANT]
+====
+Avoid using regular expressions for stream labels, as this can increase data volume.
+====
+
+=== Customizing OpenShift defaults
+
+In `openshift-logging` mode, certain attributes are required and cannot be removed from the configuration due to their role in OpenShift functions. Other attributes, labeled *recommended*, might be disabled if performance is impacted. For a complete attribute list, see xref:../../../observability/logging/logging-6.0/log6x-opentelemetry-data-model.adoc#attributes[OpenShift default attributes].
+
+When using the `openshift-logging` mode without custom attributes, you can achieve immediate compatibility with OpenShift tools. If additional attributes are needed as stream labels or structured metadata, use custom configuration. Custom configurations can *merge* with default configurations.
```

Comment:
Is *merge* a label / UI element? Or can it be plaintext:
```suggestion
When using the `openshift-logging` mode without custom attributes, you can achieve immediate compatibility with OpenShift tools. If additional attributes are needed as stream labels or structured metadata, use custom configuration. Custom configurations can merge with default configurations.
```
Emphasis doesn't seem necessary, if that's what it was for.

Reply:
It be plaintext.